packages/webpack/src/runtime/debug.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @type {Record<string, { globals: Record<string, boolean> }>}
+ */
+const incrementalPolicy = {}
+
+const printPolicyDebug = () => {
+  const policy = JSON.stringify(incrementalPolicy, null, 2)
+  console.dir(policy)
+}
+globalThis.LM_printPolicyDebug = printPolicyDebug
+
+/**
+ * @type {NodeJS.Timeout}
+ */
+let debounceTimer
+
+/**
+ * Adds a key to the incremental policy.
+ *
+ * @param {string} hint - The hint for the incremental policy.
+ * @param {string} key - The key to be added to the incremental policy.
+ * @returns {void}
+ */
+function addToPolicy(hint, key) {
+  if (!incrementalPolicy[hint]) {
+    incrementalPolicy[hint] = { globals: Object.create(null) }
+  }
+  if (!incrementalPolicy[hint].globals[key]) {
+    incrementalPolicy[hint].globals[key] = true
```

Comment:
Would `globals` make more sense as a `Set`?

Reply:
they're in a shape that resembles policy more - makes it easier to grab them from `printPolicyDebug`